packages/web-react/src/components/Checkbox/stories/CheckboxProps.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { RadioField } from '../RadioField';
```

Comment:
Interesting 🤔

Reply:
Oh wait! Are we really renaming `RadioFieldProps.tsx` to `CheckboxProps.tsx` or this is some kind of Git magic?